doc/api.rst

```diff
@@ -419,6 +550,33 @@ Reshaping and reorganizing
    DataArray.sortby
    DataArray.broadcast_like
 
+Plotting
+--------
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor.rst
+
+   DataArray.plot
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_callable.rst
+
+   DataArray.plot
```

Comment:
I need to investigate this a bit more, I can't seem to find out how to get sphinx to document `DataArray.plot()` properly. Also, we should probably only have either `accessor_callable` or `accessor`, not both.

Reply:
`DataArray.plot` is duplicated above.

To me, the summary is a minor regression compared to the major improvement --- showing that the plotting methods can be used through the accessor interface.

Reply:
yeah, sorry about that, I had those removed locally but forgot to push. The summary works for `pandas.Series.plot` so I've been trying to find out what they do differently

Reply:
should we merge this and leave the summary (and possibly the conversion to `:py:attr:` if someone complains) to a new PR?